src/main/drivers/timer_hal.c

```diff
@@ -1123,3 +1167,26 @@ HAL_StatusTypeDef DMA_SetCurrDataCounter(TIM_HandleTypeDef *htim, uint32_t Chann
     /* Return function status */
     return HAL_OK;
 }
+
+#ifdef USE_TIMER_MGMT
+uint8_t timerAlternateFunction(const timerTag_t timerTag, const ioTag_t ioTag)
```

Comment:
(There are multiple pins possible for single channel on F3)

Reply:
HAL is only used by F7 currently. I will add a wrapper.

Reply:
Jimple cjheck should do for now:
```
#ifndef STM32F7
# error "..."

Reply:
done